tools/ZAPD/ZAPD/ZSkeleton.h

```diff
@@ -2,64 +2,40 @@
 
 #include <vector>
 #include <string>
-#include <stdint.h>
+#include <cstdint>
```

Comment:
curious why this became cstdint

Reply:
youll prob want to ask either on the zap github, or the tools channel. this isnt a change I made